README.md

```diff
@@ -40,6 +40,11 @@ The DSN has to be in the following format:
 
 For further explanations have a look at the [Doctrine documentation](http://doctrine-orm.readthedocs.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url).
 
+You can also specify the buffer size with an option. Add a suffix (KB, MB or GB) to the value for better readability.
+
+Example:
+`slimdump --buffer-size 16MB {DSN} {config-file}`
```

Comment:
If the value is not optional (see below), probably `--buffer-size=16MB` would be appropriate?